src/completers/docker.zsh

```diff
@@ -7,47 +7,82 @@ _fzf_complete_docker() {
     setopt local_options no_aliases
     local command_pos=$(_fzf_complete_get_command_pos "$@")
     local arguments=("${(Qe)${(z)$(_fzf_complete_requote_arguments ${~${(z)@}})}[@][$command_pos, -1]}")
-    local subcommand=${arguments[2]}
 
     if (( $command_pos > 1 )); then
         local -x "${(Qe)${(z)$(_fzf_complete_requote_arguments ${~${(z)@}})}[@][1, $command_pos - 1]}"
     fi
 
-    if (( $+functions[_fzf_complete_docker_${subcommand}] )) && _fzf_complete_docker_${subcommand} "$@"; then
-        return
+    local options_and_subcommand subcommands
+    local docker_default_global_arguments=()
```

Comment:
What do you mean by `default` here?

Reply:
This means that arguments not from `$@` but are passed to a completion command.